master/buildbot/db/buildrequests.py

```diff
@@ -869,9 +869,11 @@ def thd(conn):
                     buildername=str(row.buildername),
                     number=row.number,
                     is_merged=bool(row.mergebrid),
+                    startbrid=row.startbrid,
                     full_name="{buildername} #{number} {merged}".format(merged=merged, **row),
                 )
                 buildrequests.append(buildrequest)
+            res.close()
```

Comment:
This `close` should be in a `try: finally:` block (or using a context manager)

This reminds me, could you create a JIRA task to add these `close` calls everywhere? That would be part of fixing this problem:

https://unity.slack.com/archives/G2Y6YTCFL/p1518439105000191

Reply:
I created: https://jira.hq.unity3d.com/browse/CDS-1832